src/constants/index.ts

```diff
@@ -1,10 +1,10 @@
+import { WXDAI, OWL_XDAI, USDC_XDAI, USDT_XDAI, WBTC_XDAI, GNO_XDAI, HONEY_XDAI } from 'utils/xdai/constants'
 import { ChainId, JSBI, Percent, Token, WETH } from '@uniswap/sdk'
 import { AbstractConnector } from '@web3-react/abstract-connector'
 
-import { fortmatic, injected, portis, walletconnect, walletlink } from '../connectors'
+import { fortmatic, injected, portis, walletconnect, walletlink } from 'connectors'
 
 export const ROUTER_ADDRESS = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
-
```

Comment:
I suggest keeping the empty line to have one less conflict to resolve when updating it